### ISSUE TITLE
wish: the space of the book / the space of the HTML doc

### ISSUE THREAD

**Issue by furter** [hybrid-publishing-lab/publishing-converter#3]
We also talked at some point about being able to save position information: sections in the book (need to be “marked” in the INDD document), position of the images in the page/spread…
What info do we need to keep from the book’s architecture that would make sense in other formats?

**Comment by gimsieke** [comment]
Yes, it will be useful to capture these coordinates. We need to extend idml2xml in order to also capture this information.
In addition, we need to specify what we should to with linked frames, where a contiguous story runs from frame to frame. Will we only output the first frame (as a div) with its position and dimension information? What do we do with the linked frames?
This is an issue that the [CSS regions](http://dev.w3.org/csswg/css-regions-1/) draft spec is probably addressing. I’m not sure whether we are already able to map these linked frames to HTML/CSS. But even if we are able to capture this information, current browsers and ebook readers are not capable of rendering it properly.
